lib/hyacinth/utils/csv_import_export_utils.rb

```diff
@@ -161,11 +164,16 @@ def self.create_import_job_from_csv_data(csv_data_string, import_filename, user)
       # Save the csv file to the filesystem
       IO.binwrite(path_to_csv_file, csv_data_string)
 
+      prerequisite_row_map = Hyacinth::Utils::CsvImportExportUtils.prerequisite_row_map_for_csv_data(csv_data_string)
+
       Hyacinth::Utils::CsvImportExportUtils.csv_to_digital_object_data(csv_data_string) do |digital_object_data, csv_row_number|
+        prerequisite_csv_row_numbers = prerequisite_row_map[csv_row_number].present? ? prerequisite_row_map[csv_row_number].map { |row_number| row_number + 1 } : [] # CSV row numbers are 1-indexed
+
         digital_object_import = DigitalObjectImport.create!(
           import_job: import_job,
           digital_object_data: JSON.generate(digital_object_data),
-          csv_row_number: csv_row_number + 1
+          csv_row_number: csv_row_number + 1, # CSV row numbers are 1-indexed
+          prerequisite_csv_row_numbers: prerequisite_csv_row_numbers
```

Comment:
Instead of saving prerequisite_csv_row_numbers, I wonder if we could store the id of the digital object import. For what I can tell it would mean that the jobs would have to be queued in a certain order in order to retrieve an id. This might add more complexity than its worth, but i figured i would throw it out there.

Reply:
I'll have to think more about this, but it would be nice not to have another field.  We should revisit this idea later on.